### PR TITLE
feat: create 2023-11-01 dotnet sdk

### PR DESCRIPTION
**Pull request by brrusino**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/chaos/Azure.ResourceManager.Chaos/src/Generated/Models/OperationStatus.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        /// <summary> The operation name. </summary>
+        public string Name { get; }
+        /// <summary> The start time of the operation. </summary>
+        public string StartTime { get; }
+        /// <summary> The end time of the operation. </summary>
+        public string EndTime { get; }
+        /// <summary> The status of the operation. </summary>
+        public string Status { get; }
+        /// <summary> The error detail of the operation if any. </summary>
+        internal ErrorResponse Error { get; }
```
Comment (brrusino) [inline sdk/chaos/Azure.ResourceManager.Chaos/src/Generated/Models/OperationStatus.cs:48]: @ArthurMa1978 We are using the common-types errorResponse, as directed, but this is creating errors that block build. Is there something we should be doing differently, or is this potentially an AutoRest bug?

Comment (brrusino) [reply sdk/chaos/Azure.ResourceManager.Chaos/src/Generated/Models/OperationStatus.cs:48]: Basically, we end up with two conflicting `Error` properties in this `OperationStatus` class.

Comment (MasterRyd3l) [reply sdk/chaos/Azure.ResourceManager.Chaos/src/Generated/Models/OperationStatus.cs:48]: the relevant swagger spec is here https://github.com/Azure/azure-rest-api-specs/blob/5ae522bc106bf8609c6cb379e584aa3e0e2639f3/specification/chaos/resource-manager/Microsoft.Chaos/stable/2023-11-01/operationStatuses.json#L113C11-L113C11

Comment (MasterRyd3l) [reply sdk/chaos/Azure.ResourceManager.Chaos/src/Generated/Models/OperationStatus.cs]: we addressed the issue. it turns out to be an issue with our openapi spec

File: sdk/chaos/Azure.ResourceManager.Chaos/tests/Scenario/ExperimentTests.cs [hunk 2]
```diff
@@ -70,23 +70,26 @@ public async Task Delete()
             var deleteResponse = await experimentResourceResponse.Value.DeleteAsync(WaitUntil.Completed).ConfigureAwait(false);
             Assert.AreEqual(200, deleteResponse.GetRawResponse().Status);
 
+            await Task.Delay(2000);
```
Comment (live1206) [inline sdk/chaos/Azure.ResourceManager.Chaos/tests/Scenario/ExperimentTests.cs:73]: Consider to use [RecordedTestBase.Delay](https://github.com/Azure/azure-sdk-for-net/blob/7eba637ebc9929a645abe3ca08b821aa349a5d52/sdk/core/Azure.Core.TestFramework/src/RecordedTestBase.cs#L504)

**Review by live1206 — COMMENTED** [review]
Please update Changelog with newly added feature and expecting release date for the new preview API version.

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**